docker/torch/deps/cuda/Dockerfile

```diff
@@ -2,17 +2,28 @@ FROM phissenschaft/xenial-cuda9-cudnn7-base
 MAINTAINER philip phissenschaft@gmail.com
 
 USER root
-# RUN apt-get update && apt-get install -y --no-install-recommends \
-#     python-dev
+RUN set -x \
+    && apt-get update && apt-get install -y --no-install-recommends \
+    # We need this for `spack` to work
+    gfortran \
+    # We want to edit stuffs in the container
+    emacs-nox \
+    sudo \
+    && rm -rf /var/lib/apt/lists/*
 
 USER drgscl
 WORKDIR /home/drgscl
 
 RUN git clone https://github.com/spack/spack.git
 
 WORKDIR spack/bin
-RUN ./spack install zlib bzip2 xz snappy readline
-RUN ./spack install lua-jit
```

Comment:
We did not need `spack` to install `lua-jit`. It will be provided by `torch` itself.